tests/test.sh

```diff
@@ -15,10 +15,8 @@ python3 -m unittest run_tests
 [[ $? -ne 0 ]] && popd && exit 1
 popd
 
-for dir in 'cbor_decode/' 'cbor_encode/' ;
-do
-        pushd "$dir"
-        ./test.sh
-        [[ $? -ne 0 ]] && popd && exit 1
-        popd
-done
+if [[ -z "$ZEHPYR_BASE" ]]; then
+        ZEPHYR_BASE=$(west topdir)/zephyr
```

Comment:
Do we need this? You are not able build zephyr samples (ie the tests) if this is not set IIRC.